extensions/azdata/src/test/azdata.test.ts

```diff
@@ -47,26 +48,251 @@ describe('azdata', function () {
 		});
 	});
 
-	// TODO: Install not implemented on linux yet
-	describe('downloadAndInstallAzdata', function (): void {
-		it('successful download & install', async function (): Promise<void> {
-			sinon.stub(childProcess, 'executeCommand').returns(Promise.resolve({ stdout: '', stderr: '' }));
-			if (process.platform === 'linux') {
-				sinon.stub(childProcess, 'executeSudoCommand').returns(Promise.resolve({ stdout: '', stderr: '' }));
+	describe('installAzdata', function (): void {
+		it('successful install', async function (): Promise<void> {
+			switch (process.platform) {
```

Comment:
I'd suggest splitting these each up into their own test that only runs on the appropriate platform. Lot easier to read.